### ISSUE TITLE
Feature Request: Alert Event Handler for AWS SNS or Kinesis

### ISSUE THREAD

**Issue by sputnik13** [influxdata/kapacitor#1494]
An AWS SNS or Kinesis handler for alert events would allow for very flexible handling of alert events by processing these events with a Lambda function.

**Comment by nathanielc** [comment]
@sputnik13 I agree this would be a good feature addition to Kapacitor. Kapacitor already has a dependency on some aws client code for its scraping feature so creating an SNS or Kinesis handler should not add too much more beyond that. 

Going to mark this feature request as help wanted for now.
